recipes-devtools/cargo/files/0001-disable-cargo-snapshot-fetch.patch

```diff
@@ -8,20 +8,20 @@ Subject: [PATCH] disable cargo snapshot fetch
  1 file changed, 4 deletions(-)
 
 diff --git a/Makefile.in b/Makefile.in
-index 286a593..9c66486 100644
+index fadcb4b..a8da630 100644
 --- a/Makefile.in
 +++ b/Makefile.in
-@@ -92,10 +92,6 @@ test-unit-$(1): $$(CARGO)
+@@ -103,10 +103,6 @@ test-unit-$(1): $$(CARGO)
  endef
  $(foreach target,$(CFG_TARGET),$(eval $(call CARGO_TARGET,$(target))))
  
--$(TARGET_ROOT)/snapshot/bin/cargo$(X): src/snapshots.txt
--	$(CFG_PYTHON) src/etc/dl-snapshot.py $(CFG_BUILD)
+-$(TARGET_ROOT)/snapshot/bin/cargo$(X): $(S)src/snapshots.txt
+-	$(CFG_PYTHON) $(S)src/etc/dl-snapshot.py $(CFG_BUILD)
 -	touch $@
```

Comment:
This is rebasing the patch that removes those lines. I assume we still need to remove those since we're fetching the snapshot with bitbake, but admittedly I haven't looked into the details of how that's set up. Perhaps it could be removed as a separate change.